pkgs/desktops/gnome/core/evolution-data-server/default.nix

```
@@ -125,10 +129,37 @@ stdenv.mkDerivation rec {
   ];
 
   passthru = {
-    updateScript = gnome.updateScript {
-      packageName = "evolution-data-server";
-      versionPolicy = "odd-unstable";
-    };
+    hardcodeGsettingsPatch =
+      runCommand
+        "hardcode-gsettings.patch"
+        {
+          inherit src;
+          nativeBuildInputs = [
+            git
+            coccinelle
+          ];
+        }
+        ''
```

```suggestion
      runCommand "hardcode-gsettings.patch" {
        inherit src;
        nativeBuildInputs = [
          git
          coccinelle
        ];
      } ''
```
or is nixpkgs-fmt undoing this?

---

Right, nixpkgs-fmt does something between these two styles so I went with the more consistent one that is also fine with nixpkgs-fmt.